client/src/components/Card/Card.stories.js

```diff
@@ -37,7 +37,7 @@ Location.args = {
   count: false,
   children: (
     <p>
-      Name <br /> Adress, <br />
+      Name <br /> Address, <br />
```

Comment:
Please don't use br. This is a visual component thing and should not be hard-coded